web/viewer.js

```diff
@@ -177,6 +177,11 @@ function getViewerConfiguration() {
 
 function webViewerLoad() {
   let config = getViewerConfiguration();
+  let locationHref = window.location.href;
+  let downloadButton = config.toolbar.download;
```

Comment:
Isn't there also a `secondaryToolbar` for the overflow menu?

Reply:
Yeah you're right.  I'll fix that too once I get a confirmation that the changes I have made is correct